src/main/scala/com/databricks/spark/avro/package.scala

```diff
@@ -44,4 +49,68 @@ package object avro {
   implicit class AvroDataFrameReader(reader: DataFrameReader) {
     def avro: String => DataFrame = reader.format("com.databricks.spark.avro").load
   }
+
+  /**
+   * Adds a method, `toRowRDD`, to RDDs of GenericRecords.  This allows you to convert it to
+   * an RDD of Row (and, by implication, a DataFrame).
+   */
+  implicit class GenericRecordRDD(rdd: RDD[GenericRecord]) {
+    def toRowRDD(requiredColumns: Array[String]): RDD[Row] = rdd.mapPartitions { records =>
```

Comment:
Could we instead make this `toDF(): DataFrame`?  I think that gives us more long term flexibility, for example, if we wanted to skip the public row representation and go directly to the internal row representation.  It also keeps the user from manually having to handle the schema when doing the conversion.